Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM alpine:latest
 
 # Version of Radicale (e.g. 2.0.0)
-ENV VERSION master
+ARG VERSION=master
+ENV VERSION ${VERSION}
```

Comment:
I don't think that ``ARG ...`` and ``ENV ...`` are both required. Just ``ARG ...`` should be enough.